lab-tuple-set-dict/your-code/.ipynb_checkpoints/challenge-3-checkpoint.ipynb

```diff
@@ -0,0 +1,587 @@
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'a': 8, 'out': 1, 'about': 1, 'all': 1, 'although': 3, 'and': 23, 'are': 1, 'at': 1, 'baby': 14, 'backseat': 1, 'eat': 1, 'bag': 1, 'bar': 1, 'be': 16, 'bedsheets': 3, 'begin': 1, 'in': 27, 'i': 6, 'best': 1, 'body': 17, 'boy': 2, 'brand': 6, 'can': 1, 'chance': 1, 'club': 1, 'come': 37, 'me': 10, 'conversation': 1, 'on': 40, 'crazy': 2, 'dance': 1, 'date': 1, 'day': 6, 'over': 1, 'is': 5, 'discovering': 6, 'do': 3, 'doing': 2, \"don't\": 2, 'drinking': 1, 'driver': 1, 'every': 6, 'falling': 3, 'family': 1, 'fast': 1, 'fill': 2, 'find': 1, 'first': 1, 'follow': 6, 'for': 3, 'friends': 1, 'get': 1, 'girl': 2, 'give': 1, 'go': 2, 'going': 1, 'grab': 2, 'hand': 1, 'handmade': 2, 'heart': 3, 'our': 1, 'hours': 2, 'how': 1, \"i'll\": 1, \"i'm\": 23, \"isn't\": 1, 'it': 1, 'jukebox': 1, 'just': 1, 'kiss': 1, 'now': 11, 'know': 2, 'last': 3, 'lead': 6, 'leave': 1, 'let': 1, \"let's\": 2, 'like': 10, 'love': 25, 'lover': 1, 'magnet': 3, 'make': 1, 'man': 1, 'may': 2, 'mind': 2, 'much': 2, 'my': 33, 'new': 6, 'night': 3, 'not': 2, 'of': 6, 'okay': 1, 'one': 1, 'place': 1, 'plate': 1, 'play': 1, 'pull': 3, 'push': 3, 'put': 3, 'radio': 1, 'room': 3, 'say': 2, 'shape': 6, 'shots': 1, 'singing': 2, 'slow': 1, 'smell': 3, 'so': 2, 'somebody': 2, 'something': 6, 'sour': 1, 'start': 2, 'to': 2, 'stop': 1, 'story': 1, 'we': 7, 'sweet': 1, 'table': 1, 'take': 1, 'talk': 4, 'taxi': 1, 'tell': 1, 'that': 2, 'the': 18, 'then': 3, 'thrifty': 1, 'too': 5, 'trust': 1, 'up': 3, 'van': 1, 'waist': 2, 'want': 2, 'was': 2, \"we're\": 1, 'week': 1, 'were': 3, 'where': 1, 'with': 22, 'you': 16, 'your': 21}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Your code here\n",
+    "word_freq2={}\n",
+    "keys= list(word_freq.keys())\n",
+    "keys.sort()\n",
+    "\n",
+    "for k in keys:\n",
+    "    for a, b in word_freq.items():\n",
+    "        if a in k:\n",
+    "            word_freq2[a]=b\n",
+    "print(word_freq2)\n",
```

Comment:
Puedes también usar: 
for k in keys:
    word_freq2[key] = word_freq[key]
print(word_freq2)